### ISSUE TITLE
TypeError: extract_FES_constants() got multiple values for argument 'TYPE'

### ISSUE THREAD

**Issue by alexguyot** [tsutterley/pyTMD#50]
Hello,

Thanks again for the great work with this package !

I'm using pyTMD from a pip install.
Recently, I installed the latest version :  

`pip install pyTMD==1.0.2.18`

But an error is now raised when extracting FES constants :
```
---------------------------------------------------------------------------`
TypeError                                 Traceback (most recent call last)
<ipython-input-3-f7f0e7858d5d> in <module>
     39 #-- read tidal constants and interpolate to grid points
     40 if (model_format == 'FES'):
---> 41     amp,ph = extract_FES_constants(np.array([LON]), np.array([LAT]), model_directory, model_files, TYPE=TYPE, VERSION=TIDE_MODEL, METHOD='spline', SCALE=SCALE)
     42 
     43     #-- interpolate delta times from calendar dates to tide time
TypeError: extract_FES_constants() got multiple values for argument 'TYPE'
```
I reverted back to the previous version `pip install pyTMD==1.0.2.17` : works fine.

Alex

**Comment by tsutterley** [comment]
Sorry @alexguyot.  Totally my fault.  I should have written this in a way that was backwards compatible (or at least added deprecation warnings before I changed things).  I changed the arguments to the FES read program in 1.0.2.18 so that there would be more uniformity across the read programs (OTIS binary, OTIS netCDF, GOT, and FES).  I removed the `model_directory` argument and made `model_files` absolute paths.  I used the following lines to make the absolute paths
```python
model_file = [os.path.join(model_directory,m) for m in model_files]
amp,ph = extract_FES_constants(np.array([LON]), np.array([LAT]), model_file, \
   TYPE=TYPE, VERSION=TIDE_MODEL, METHOD='spline', SCALE=SCALE)
```
Let me know how it goes.
